### DISCUSSION TITLE
Why is astropy a requirement?

### DISCUSSION THREAD

**Discussion by claytonjr** [dcajasn/Riskfolio-Lib#209] [Q&A]
Was just looking at the requirements file, and noticed astropy in there. That's a library for machine learning and astronomy, right? It just seems like an odd requirement choice for a library for financial risk modelling.

**Comment by dcajasn** [comment]
Hi @claytonjr,

It is required to determine de optimal bin width when we use mutual information as codependence matrix.

Best,
Dany
